### DISCUSSION TITLE
Please clarify message "Error loading memory: ASM Emitting unfilled A instruction"

### DISCUSSION THREAD

**Discussion by CodeQuiver** [DavidSouther/computron5k#536] [General]
Hi! I'm working on Project 7 and have encountered this error when trying to upload my StackTest.asm translated file to the CPU Emulator. Can a contributor clarify what specific type of code error this is referring to?

I'm sure my code legitimately has an error, but even after tracing the message and the code that triggers it in the repo, I can't tell exactly what "Emitting unfilled A instruction" is meant to indicate. I don't have any lines that are just "@" with nothing else, which would have been my first guess. Debugging would be much easier if I understood the error. Thanks!

Edit: This appears to show up when the same label is used more than once, but I'm not sure if that's what's intended to be checked or if the error check was for something different but related. Example code:
```
@EQ_TRUE
D;JEQ
D=0
@EQ_SAVE
0;JMP
(EQ_TRUE)
D=-1
(EQ_SAVE)
...
@EQ_TRUE
D;JEQ
D=0
@EQ_SAVE
0;JMP
(EQ_TRUE)
D=-1
(EQ_SAVE)
...
```

**Comment by technomancy** [comment]
I also found that this error message is mistakenly emitted when you emit a label which is predefined; for example `(R1)` will trigger it since `R1` is already the name of an existing register.

It would be great if the error message could be fixed, but it would be even *better* if error messages showed the line number of the error. Debugging this without line numbers took an awful lot of bisecting.
